spec/controllers/attributes_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require Rails.root.join('spec', 'support', 'mock_auth_helper.rb')
+
+describe AttributesController do
```

Comment:
I don't see why I shouldn't test controllers.  They're part of the application, so they should be tested.  I don't want to write integration tests to test these really simple controllers...